src/genie/libs/parser/iosxe/tests/ShowDeviceTrackingDatabaseInt/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,836 @@
+expected_output = {
+    "binding_table": {"entries": 87, "dynamic": 75},
```

Comment:
please add `'limit': 100000`.
```suggestion
    "binding_table": {"entries": 87, "dynamic": 75, "limit": 100000},
```

Reply:
Done!